zproject/local_settings_template.py

```diff
@@ -298,3 +298,22 @@
 }
 
 CAMO_URI = ''
+
+# By default, Zulip connects to rabbitmq running locally on the machine,
+# but Zulip also supports connecting to RabbitMQ over the network;
+# to use a remote RabbitMQ instance, set RABBITMQ_HOST here.
+# RABBITMQ_HOST = "localhost"
+# To use another rabbitmq user than the default 'zulip', set RABBITMQ_USERNAME here.
+# RABBITMQ_USERNAME = 'zulip'
+
+# Memcached Server address, by default.
```

Comment:
Probably the memcached comment here should be similar to the redis/rabbitmq ones.